scripts/data/experiment.json

```diff
@@ -28,6 +28,53 @@
             {"type": "audio/ogg", "src": "horse.ogg"}
         ]
     }, {
+        "id": "one-randomizer",
+        "type": "random-choice",
+        "options": [{
+            "id": "maybe-consent",
+            "type": "consent",
+            "title": "Just checking",
+            "body": "Are you sure you know what you're getting into?",
+            "consentLabel": "I was told there were cookies"
+        }, {
+            "id": "maybe-soundcheck",
```

Comment:
If I failed to understand or articulate how I expected this to work, my apologies. It's still unclear how this implies "dependence", but we may just be using different vocabulary. 

I was hoping the sample structure indicated in https://openscience.atlassian.net/browse/LEI-89 would help avoid confusion. My hope was that each branch defined in "options" would be a list rather than a single item.